index.html

```diff
@@ -0,0 +1,931 @@
+<!DOCTYPE html>
+<html lang="en">
+<meta charset="UTF-8">
+<meta name="viewport" content="width= 320, initial-scale= 0.5">
+<meta http-equiv="X-UA-Compatible" content="ie=edge">
+<title>NEWYORKTIMES PAGE</title>
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css">
+<link rel="stylesheet" href="./assets/style/style.css">
+
+<body>
+
+    <header class="menu-bar">
+        <div class="header-container">
```

Comment:
The `.header-container` should be a <`nav`> tag and not a div, you should try and write proper HTML semantic code, also the navigation bar should have a fixed position (it sticks to the top of the screen)([link to the page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))